db/migrate/20170613144708_create_openshift_image.rb

```diff
@@ -0,0 +1,21 @@
+class CreateOpenshiftImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Adding type and ems_ref columns to ContainerImage and setting type") do
+      add_column :container_images, :type, :string
+      ContainerImage.update_all(:type => "ManageIQ::Providers::Openshift::ContainerManager::ContainerImage")
+      ContainerImage.where(:author => nil).update_all(:type => "ContainerImage")
+      add_column :container_images, :ems_ref, :string
```

Comment:
Do we have a value to update for the Openshift::ContainerManager::ContainerImage ?